fdb-record-layer-core/src/test/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexerBuildSumIndexTest.java

```diff
@@ -103,6 +103,9 @@ public void run() {
                 try (FDBRecordContext context = openContext()) {
                     long sum = recordStore.evaluateAggregateFunction(Collections.singletonList("MySimpleRecord"), aggregateFunction, TupleRange.ALL, IsolationLevel.SNAPSHOT).join().getLong(0);
                     long expected = updatedRecords.stream().mapToInt(msg -> msg.hasNumValue2() ? msg.getNumValue2() : 0).sum();
+                    for (int i = 0; sum != expected && i < 20; i ++) {
+                        Thread.yield();
```

Comment:
Hm, what is this line trying to accomplish?

Reply:
This meaningless code made the sum test pass, I do not understand why. 
After distinguishing between idempotent and accumulative indexes, the sum test passes without this ghost hunting.